src/tests/multihost/ad/test_adparameters.py

```diff
@@ -475,29 +475,26 @@ def test_0013_bz1794016(self, multihost, adjoin):
         :id: ac5e99cc-2d81-48f7-82ff-622f1c6b3684
```

Comment:
`files provider is also enabled` -> `multiple domains are configured`

Reply:
Done